vnext/PropertySheets/JSEngine.props

```diff
@@ -2,8 +2,8 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <UseHermes Condition="'$(UseHermes)' == ''">false</UseHermes>
-    <HermesVersion Condition="'$(HermesVersion)' == ''">0.7.2-microsoft.5</HermesVersion>
+    <UseHermes Condition="'$(UseHermes)' == ''">true</UseHermes>
```

Comment:
this seems like it is turning Hermes on by default, do we have a writeup of what is working/what's missing, is the tooling ready, etc?

Reply:
I'll work on a writeup ..

Reply:
This change to globally enable hermes was unintentional .. Reverted.